src/nvim/eval/typval.h

```diff
@@ -352,6 +526,75 @@ extern const char *const tv_empty_string;
 /// Specifies that free_unref_items() function has (not) been entered
 extern bool tv_in_free_unref_items;
 
+/// Iterate over a list
+///
+/// @param  modifier  Modifier: expected to be const or nothing, volatile should
+///                   also work if you have any uses for the volatile list.
+/// @param[in]  l  List to iterate over.
+/// @param  li  Name of the variable with current listitem_T entry.
+/// @param  code  Cycle body.
```

Comment:
Perhaps use loop instead of cycle?

Reply:
Why? They are synonims and that was already used for TV_DICT_ITER. And loop is also commonly used as a shortcut for “event loop” which has nothing to do with _ITER macros.